### PR TITLE
Fix the DPLA audiobook download issue

### PR DESCRIPTION
**Pull request by vbessonov**
## Description

This PR adds changes required to extract audiobooks' content type and DRM scheme from LCP License Info Document and use it to create a correct delivery mechanism.

## Checklist:

- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jonathangreen** [comment]
Can you remove the changes from https://github.com/ThePalaceProject/circulation-core/pull/40/commits/ca418dc479db0ea4d3e0139f9cb65d0f58a86040 in this PR? Seems that commit is not related here.

File: opds_import.py [hunk 1]
```diff
@@ -65,10 +65,14 @@ def parse_identifier(db, identifier):
     :rtype: Optional[core.model.identifier.Identifier]
     """
     parsed_identifier = None
-    result = Identifier.parse_urn(db, identifier)
 
-    if result is not None:
-        parsed_identifier, _ = result
+    try:
+        result = Identifier.parse_urn(db, identifier)
```
Comment (jonathangreen) [inline opds_import.py:70]: Are these changes related to the DPLA download issue? When were we seeing an exception parsing an identifier?

Comment (vbessonov) [reply opds_import.py:70]: Yes, they're related. When I was testing my changes I saw at least one DPLA identifier that wasn't a valid ISBN even though it claimed to be (by using `urn:ISBN` scheme).

**Comment by jonathangreen** [comment]
Looks good to me other then the unrelated `lcp:hashed_passphrase` changes.

**Review by jonathangreen — APPROVED** [review]
